manifests/configmap.yaml

```diff
@@ -28,7 +28,7 @@ data:
   db_hosted_zone: db.example.com
   master_dns_name_format: '{cluster}.{team}.staging.{hostedzone}'
   replica_dns_name_format: '{cluster}-repl.{team}.staging.{hostedzone}'
-  enable_master_load_balancer: "false"
+  enable_master_load_balancer: "true"
```

Comment:
Why does it make sense to set its default to true when aiming for minikube users?

Reply:
we need the LB service of such type here for it to be [accessible outside of minikube](https://github.com/kubernetes/minikube/issues/384#issuecomment-234409957).